cmd/rook/ceph/osd.go

```diff
@@ -275,9 +275,9 @@ func commonOSDInit(cmd *cobra.Command) {
 // Parse the devices, which are comma separated. A colon indicates a non-default number of osds per device
 // or a non collocated metadata device.
 // For example, one osd will be created on each of sda and sdb, with 5 osds on the nvme01 device.
-//   sda:1,sdb:1,nvme01:5
+//   sda:1::,sdb:1::,nvme01:5::
```

Comment:
Can we add this in the documentation too?

Reply:
I only see this: https://rook.io/docs/rook/v1.0/ceph-cluster-crd.html#osd-configuration-settings which does not describe the `sda:1::,sdb:1::,nvme01:5::` behavior, or am I missing something?